cluster/service.go

```diff
@@ -244,30 +243,19 @@ func (s *Service) processWriteShardRequest(buf []byte) error {
 	s.statMap.Add(writeShardPointsReq, int64(len(points)))
 	err := s.TSDBStore.WriteToShard(req.ShardID(), points)
```

Comment:
What are you doing with this err value?  I don't see it being checked any more.  If it succeeded, it should return right?